plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/FlushMetadataCacheProcedure.java

```diff
@@ -98,14 +102,17 @@ private void doFlushMetadataCache(ConnectorSession session, Optional<String> sch
         if (schemaName.isEmpty() && tableName.isEmpty()) {
             cachingHiveMetastore.ifPresent(CachingHiveMetastore::flushCache);
             transactionLogAccess.flushCache();
+            extendedStatisticsAccess.invalidateCache();
         }
         else if (schemaName.isPresent() && tableName.isPresent()) {
             HiveMetastore metastore = metastoreFactory.createMetastore(Optional.of(session.getIdentity()));
             Table table = metastore.getTable(schemaName.get(), tableName.get())
                             .orElseThrow(() -> new TableNotFoundException(new SchemaTableName(schemaName.get(), tableName.get())));
             verifyDeltaLakeTable(table);
             cachingHiveMetastore.ifPresent(caching -> caching.invalidateTable(table.getDatabaseName(), table.getTableName()));
-            transactionLogAccess.invalidateCaches(getTableLocation(table));
+            String tableLocation = getTableLocation(table);
+            transactionLogAccess.invalidateCaches(tableLocation);
+            extendedStatisticsAccess.invalidateCache(tableLocation);
```

Comment:
nit: Is it possible to add a test for this change? Or do you think `TestDeltaLakeFileOperations` is sufficient?

Reply:
it would be hard to have an explicit test for that. yes, TestDeltaLakeFileOperations covers that (implicitly)